server/src/schemas/auditLogsSchema.ts

```diff
@@ -3,10 +3,10 @@ import { gql } from "apollo-server-express";
 export const AuditLogsTypeDefs = gql`
   type Log {
     id: ID!
-    timeDate: Date!
+    timeDate: DateTime!
```

Comment:
for this to work you need to import the DateTime scalar, see `machinesSchema` or `storeFrontSchema` maybe for reference